pkg/scheduler/eventhandlers.go

```diff
@@ -225,7 +227,10 @@ func (sched *Scheduler) updatePodInCache(oldObj, newObj interface{}) {
 		logger.Error(err, "Scheduler cache UpdatePod failed", "pod", klog.KObj(oldPod))
 	}
 
-	sched.SchedulingQueue.AssignedPodUpdated(logger, oldPod, newPod)
+	events := podSchedulingPropertiesChange(newPod, oldPod)
+	for _, evt := range events {
```

Comment:
> Although in theory, it shouldn't happen more than once, as it is unlikely that a single update would change both requests and labels.

I don't concern very much because of this reason.

But, eventually it's better to aggregate all changes in the update and send only one aggregated event for SchedulingQueue. However it's not that simple to do so right away because some functions in SchedulingQueue use event.Label for metric label and logging. So, I'd like to keep here as it is, and do them in another PR as a follow up.